hotham/src/systems/audio.rs

```diff
@@ -121,7 +121,7 @@ mod tests {
     };
 
     use rapier3d::prelude::{RigidBodyBuilder, RigidBodyType};
-    const DURATION_SECS: u32 = 8;
+    const DURATION_SECS: u32 = 3;
```

Comment:
This looks unrelated

Reply:
Just trimming the tests down! 😄 A bit unrelated, but since the whole test suite was running like kludge, I felt like this was fair.